core/Core.php

```diff
@@ -85,10 +85,11 @@
 // Initialise the dependency injector as soon as possible, as it is 
 // subsequently used by some of the following code
 $injector = new Injector(array('locator' => 'SilverStripeServiceConfigurationLocator'));
-$injector->registerService(Config::inst());
-
 Injector::set_inst($injector);
 
+// Initialise Config
+Config::inst();
```

Comment:
I'm not sure why this is required as anywhere else that uses config will have to call this?